lidar_lite_v3hp/launch/lidar_litev3hp.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+  <node pkg="rosserial_python" type="serial_node.py" name="serial_node1">
+    <param name="port" value="/dev/ttyACM0"/>
+    <param name="baud" value="57600"/>
+  </node>
+  <node pkg="rosserial_python" type="serial_node.py" name="serial_node2">
+    <param name="port" value="/dev/ttyACM1"/>
+    <param name="baud" value="57600"/>
```

Comment:
The port and baud parameters are configuration parameters and should be launch file arguments.